capa/features/extractors/dnfile/helpers.py

```diff
@@ -258,10 +263,7 @@ def get_dotnet_properties(pe: dnfile.dnPE) -> Iterator[DnType]:
 
 def get_dotnet_managed_method_bodies(pe: dnfile.dnPE) -> Iterator[Tuple[int, CilMethodBody]]:
     """get managed methods from MethodDef table"""
-    if not hasattr(pe.net.mdtables, "MethodDef"):
-        return
-
-    for (rid, row) in enumerate(pe.net.mdtables.MethodDef):
+    for (rid, row) in enumerate(iter_dotnet_table(pe, "MethodDef")):
```

Comment:
we should possibly make these table names an enum, that way we can avoid typos and better document which values are appropriate here. but i think thats outside the scope of this PR.

Reply:
updated type checking - see most recent commits